lib/mayaUsd/resources/ae/usdschemabase/lightCustomControl.py

```diff
@@ -3,40 +3,72 @@
 
 class LightLinkingCustomControl(object):
     '''Custom control for the light linking data we want to display.'''
-    def __init__(self, item, prim, useNiceName):
+
+    @staticmethod
+    def isCollectionAttribute(aeTemplate, attrName):
+        '''
+        Verify if the given attribute name is a collection attribute.
+        '''
+        attrNameParts = attrName.split(':')
+        if len(attrNameParts) <= 1:
+            return False
+        
+        instanceName = attrNameParts[1]
+        if not aeTemplate.prim.HasAPI(Usd.CollectionAPI, instanceName):
+            return False
+        
+        if len(attrNameParts) > 2:
```

Comment:
This suppress the collection sub-attributes since the collection section handles all aspects of the collection.